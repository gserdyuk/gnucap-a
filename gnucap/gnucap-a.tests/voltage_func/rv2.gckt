* usage of v() in ac

.param a1='0.5+abs(v(1))'

v1 1 0 dc 2.1 ac 1
r1 1 0 10k
v2 2 0 dc 1 ac 'a1'
r2 2 0 10k


.print op v(1) v(2)
.op
.print ac v(1) v(2)
.ac dec 2 10 10K


.end


