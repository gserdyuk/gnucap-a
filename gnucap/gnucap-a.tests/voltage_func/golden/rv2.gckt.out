Gnucap gnucap-arails  2009.12.07 RCS 26.136 master 2010-09-25  bmm_semi_hsp, bm_model fix, back to bmm_semi
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
* usage of v() in ac 
#           v(1)       v(2)      
 27.        2.1        1.        
#Freq       v(1)       v(2)      
 10.        1.         0.5       
 31.623     1.         0.5       
 100.       1.         0.5       
 316.23     1.         0.5       
 1.K        1.         0.5       
 3.1623K    1.         0.5       
 10.K       1.         0.5       
